seqr/views/utils/pedigree_info_utils.py

```diff
@@ -437,7 +438,7 @@ def _parse_datstat_export_format(rows):
 
 
 def _get_datstat_family_notes(row):
-    row = {k: unicode(v, errors='ignore') for k, v in row.items()}
+    row = {k: str(v, 'utf-8', errors='ignore') if not isinstance(v, str) else v for k, v in row.items()}
```

Comment:
I don't like the conditional with the  type checking here - if it is an instance of `str` theres no harm wrapping it again and its much easier to read

Reply:
Also, this changes the output of the report in a completely unacceptable way

Reply:
`str(v, errors='ignore')` is a decoding. Python 3 can't decode `str` objects. That is the reason the conditional check is used.